Mathlib/RingTheory/AdjoinRoot.lean

```diff
@@ -716,7 +714,6 @@ def quotAdjoinRootEquivQuotPolynomialQuot :
       ((Ideal.quotEquivOfEq (by rw [map_span, Set.image_singleton])).trans
         (Polynomial.quotQuotEquivComm I f).symm))
 
--- Porting note: mathlib3 proof was a long `rw` that timeouts.
```

Comment:
Pointing this out to maintainers: I'd appreciate a look if there's any downside to this particular proof by `rfl`.